examples/rn-connection-and-error/backend/functions/deleteCustomUserDataDoc.js

```diff
@@ -0,0 +1,23 @@
+/* eslint-disable */
+
+/**
+ * Deletes a custom user data (`Users`) document.
+ */
+exports = async function deleteCustomUserDataDoc({ user }) {
```

Comment:
Added this to make it easier when playing around with the example and not needed to manually go and delete docs. Is probably good to showcase this as well to show what's reasonable to do in production.